### ISSUE TITLE
Update header includes to use files in drake/multibody/

### ISSUE THREAD

**Issue by amcastro-tri** [RobotLocomotion/director#363]
PR RobotLocomotion/drake#4058  moves all the RBT related files into `drake/multibody`.
To avoid breaking `Director`, PR RobotLocomotion/drake#4058 creates the following dummy files:

- drake/systems/plants/RigidBodyTree.h
- drake/systems/plants/ForceTorqueMeasurement.h
- drake/systems/plants/shapes/Geometry.h

so that the non-updated version of Director can still work. These files merely include the files in `drake/multibody`.

Post RobotLocomotion/drake#4058 two PR's should be submitted:

1. PR in Director updating includes to use RBT related files in `drake/multibody`
2. PR in Drake removing the dummy files cited above.

**Comment by patmarion** [comment]
I don't think it's necessary to stage things this way.  You can just develop your changes to director in a branch and your drake branch can reference it (instead of director/master).  Then when your changes are reviewed and ready to merge into drake, we merge the director PR, then update your drake branch to reference it.

**Comment by amcastro-tri** [comment]
Thank you @patmarion, this seems like a better idea. I pushed #364. PTAL.

**Comment by amcastro-tri** [comment]
After #364 Director now includes files from `drake/multibody` and thus this issue is closed.
